### PR TITLE
add 5-byte large disk builds for arm and arm64

### PR DESCRIPTION
**Pull request by mwpeterson**
I have Odroid HC2s with 12TB drives in them, so running the 5-byte build of SeaweedFS seemed correct.

### CODE REVIEW

**Comment by chrislusf** [comment]
Do you also want to add it to the release?

File: Makefile [hunk 1]
```diff
@@ -55,6 +55,14 @@ release: deps windows_build darwin_build linux_build bsd_build 5_byte_linux_buil
 	$(call build_large,windows,amd64,.exe)
 	$(call zip_large,windows,amd64,.exe)
 
+5_byte_arm_build: $(sources)
```
Comment (chrislusf) [inline Makefile:58]: 32bit arm may not be that useful

File: Makefile [hunk 2]
```diff
@@ -55,6 +55,14 @@ release: deps windows_build darwin_build linux_build bsd_build 5_byte_linux_buil
...
+	$(call build_large,linux,arm,)
+	$(call tar_large,linux,arm)
+
+5_byte_arm64_build: $(sources)
```
Comment (chrislusf) [inline Makefile:62]: you can add this to line 43, so it can be part of the release process.

File: Makefile (hunk 1 above)
Comment (mwpeterson) [reply Makefile:58]: I've spent the morning with google trying to figure out why, but I don't think I have. :(
My guess is either:

- something with bit-shifting by 32 being undefined on 32-bit
- `[]byte` can only hold 4 bytes on 32-bit

I'm guessing large disk weed will work until it needs to use that 5th byte and then it'll panic or worse.

**Pull request merged**